o-fish-ios/Views/PatrolBoat/PatrolSummary/PatrolSummaryView.swift

```diff
@@ -62,9 +59,9 @@ struct PatrolSummaryView: View {
             VStack(spacing: .zero) {
                 CallToActionButton(title: "Go Off Duty",
                     action: self.goOffDutyClicked)
-                    .padding(.top, Dimensions.Button.topPadding)
-                    .padding(.bottom, Dimensions.Button.bottomPadding)
-                    .padding(.horizontal, Dimensions.Button.leadingTrailingPadding)
+                    .padding(.top, Dimensions.buttonTopPadding)
```

Comment:
not sure about this, for me nested enum for specific configuration has better readability

Reply:
We should change this or add it to the exceptions in our SwiftLint file – either is OK with me, but we should get rid of the build warnings as they may mask something more serious

Reply:
lets keep it for this PR, I will take a look in next PR for swiftlint update